### PR TITLE
Adds integrate with editors page

### PR DESCRIPTION
**Pull request by schultzp2020**
## What does this PR do / why we need it

Adds documentation on how to get intellisense for devfiles.

## Which issue(s) does this PR fix

Fixes https://github.com/devfile/api/issues/820

## PR acceptance criteria

- [ ] Unit Tests
- [ ] E2E Tests
- [x] Documentation

### CODE REVIEW

File: libs/docs/src/docs/no-version/integrate-with-editors.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Integrate with editors
+description: Integrate with editors
+---
+
+The [YAML Language Server](https://github.com/redhat-developer/yaml-language-server) provides validation, document outlining, auto completion, hover support, and formatting for YAML files. To provide intellisense for YAML files, or more specifically devfiles, the YAML Language Server pulls all available schemas from the [JSON Schema Store](https://www.schemastore.org/json/). The JSON Schema Store contains the devfile JSON Schema, which is maintained by the devfile team.
```
Comment (mike-hoang) [inline libs/docs/src/docs/no-version/integrate-with-editors.md:6]:
```suggestion
The [YAML Language Server](https://github.com/redhat-developer/yaml-language-server) provides validation, document outlining, auto-completion, hover support, and formatting for YAML files. To provide IntelliSense for devfiles, the YAML Language Server pulls all available schemas from the [JSON Schema Store](https://www.schemastore.org/json/). The devfile team maintains the devfile JSON Schema stored within the JSON Schema Store.
```

**Review by mike-hoang — APPROVED** [review]
small changes, everything else looks good

**Comment by johnmcollier** [comment]
@gabriel-rh would you be able to review this?

**Review by mike-hoang — APPROVED** [review]

File: libs/docs/src/docs/no-version/integrate-with-editors.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+If you are using [VSCode](https://code.visualstudio.com/), you can install the [YAML VSCode Plugin](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) from the Marketplace. The YAML VSCode Plugin is built and maintained by Red Hat.
+
+### Other Editor Plugins
+
+Other editor plugins can be found in the YAML Language Server [GitHub Repository](https://github.com/redhat-developer/yaml-language-server#clients).
+
+## Access the latest version
+
+Press `Ctrl + Shift + P` and type `Preferences: Open User Settings (JSON)`. This should open up your user settings. Inside the user settings add the following snippet.
```
Comment (gabriel-rh) [inline libs/docs/src/docs/no-version/integrate-with-editors.md:24]:
```suggestion
Press `Ctrl + Shift + P` and type `Preferences: Open User Settings (JSON)`. This opens your user settings. Inside the user settings, add the following snippet:
```

**Review by gabriel-rh — APPROVED** [review]
/lgtm

Some very minor suggestions, but not essential.

**Review by johnmcollier — APPROVED** [review]

**Pull request merged**